python-client/trustedanalytics/core/files.py

```diff
@@ -701,5 +701,58 @@ def __validate(self, schema):
         return True
 
 
+class JdbcTable(object):
+    """
+    Define the object to retrieve the data from an jdbc table.
+
+    Parameters
+    ----------
+    table_name : str
+        the table name
+    url : str
+        Jdbc connection string (as url)
```

Comment:
should provide more description of the "url" format  --I can see the details in the example below